tests/e2e/kcs/test_selinux_relabel_solution.py

```diff
@@ -0,0 +1,322 @@
+            f' -c "find {data_path} -type f | "shuf" -n {num_of_files}"',
+            timeout=300,
+        )
+        random_files = random_files.split()
+        log.info(f"files are {random_files}")
+        initial_md5sum = []
+        for file_path in random_files:
+            md5sum = res_pod.cal_md5sum(
+                pod_obj=self.pod_obj,
+                file_name=file_path,
+            )
+            initial_md5sum.append(md5sum)
+
+        # Delete pod and Get time for pod restart
+        self.pod_obj.delete(wait=True)
+        self.pod_obj = self.get_app_pod()
+        try:
+            wait_for_pods_to_be_running(
+                pod_names=[self.pod_obj.name], timeout=600, sleep=15
+            )
+        except CommandFailed:
+            log.exception(f"Pod {self.pod_obj.name} didn't reach to running state")
+
+        pod_restart_time_before_fix = self.get_pod_start_time(
+            pod_name=self.pod_obj.name
+        )
+        log.info(f"Time taken by pod to restart is {pod_restart_time_before_fix}")
+
+        # Apply the fix/solution for “Existing PVCs”
+        self.apply_selinux_solution_on_existing_pvc(self.pvc_obj)
+
+        # Delete pod so that fix will be applied for new pod
+        self.pod_obj = self.get_app_pod()
+        self.pod_obj.delete(wait=True)
+        self.pod_obj = self.get_app_pod()
+        assert wait_for_pods_to_be_running(
+            pod_names=[self.pod_obj.name], timeout=600, sleep=15
+        ), f"Pod {self.pod_obj.name} didn't reach to running state"
+
+        # Get the node of cephfs pod
+        self.pod_obj = self.get_app_pod()
+        node_name = res_pod.get_pod_node(pod_obj=self.pod_obj).name
+
+        # Check SeLinux Relabeling is set to false
+        oc_cmd = ocp.OCP(namespace=constants.OPENSHIFT_STORAGE_NAMESPACE)
+        cmd1 = "crictl inspect $(crictl ps --name perf -q)"
+        output = oc_cmd.exec_oc_debug_cmd(node=node_name, cmd_list=[cmd1])
+        key = '"selinuxRelabel": false'
+        assert key in output, f"{key} is not present in inspect logs"
+        log.info(f"{key} is present in inspect logs of application pod running node")
+        log.info(f"SeLinux Relabeling is not happening for the pvc {self.pvc_obj.name}")
+
+        # Get time for pod restart after applying fix
+        self.pod_obj = self.get_app_pod()
+        self.pod_obj.delete(wait=True)
+        self.pod_obj = self.get_app_pod()
+        assert wait_for_pods_to_be_running(
+            pod_names=[self.pod_obj.name], timeout=600, sleep=15
+        ), f"Pod {self.pod_obj.name} didn't reach to running state"
+        pod_restart_time_after_fix = self.get_pod_start_time(pod_name=self.pod_obj.name)
+        log.info(f"Time taken by pod to restart is {pod_restart_time_after_fix}")
+
+        assert (
+            pod_restart_time_before_fix > pod_restart_time_after_fix
+        ), "Time taken for pod restart after fix is more than before fix."
+
+        # Check data integrity.
```

Comment:
this step is important , and we should check this first followed by the pod restart time